src/shared/components/community/sidebar.tsx

```diff
@@ -130,9 +130,13 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
             </a>
           )}
           {subscribed == SubscribedType.Pending && (
-            <div class="badge badge-warning mr-2">
+            <a
+              class="btn btn-warning mr-2"
+              href="#"
+              onClick={linkEvent(this, this.handleUnsubscribe)}
+            >
               {i18n.t("subscribe_pending")}
-            </div>
+            </a>
```

Comment:
This works for now, at some point tho I need to change all these a tags to `<button`s, a tags should only be used for links.